### ISSUE TITLE
Extraction behavior is not correct when paring the react file in <Trans> component.

### ISSUE THREAD

**Issue by purepennons** [i18next/i18next-parser#80]
- Is this a bug or a feature request?
  - bug
- What version are you running (`0.x` or `1.x`)? 
  - 1.x (1.0.0-beta2)
- Which runner (cli, gulp...)?
  - cli
## What I'm trying to do?
I try to extract json files from react-i18next examples ([react_withHOC](https://github.com/i18next/react-i18next/tree/master/example/react_withHOC) & [react_renderProps](https://github.com/i18next/react-i18next/tree/master/example/react_renderProps))

## How am I doing it?
Here is my config file:

```
// i18next-parser.config.js
module.exports = {
  reactNamespace: true,
  locales: ['en', 'de'],
  output: './locales'
};
```

The command I'm running:

```
i18next "src/**/*.{js,jsx}" -c ./i18next-parser.config.js
```

## What was the result?
both `en` and `de` are
```json
// translation_old.json
{}

// translation.json
{
  "title": "",
  "description": {
    "part2": ""
  }
}
```
## What was I expecting?
Need to extract the text from `<Trans>` too. But `description.part1` key is missing.

```json
{
  "title": "",
  "description": {
    "part1": "",
    "part2": ""
  }
}
```

**Comment by karellm** [comment]
@purepennons I think it's because the lexers are not properly configured. Can you try adding this to your `i18next-parser.config.js`:

```js
module.exports = {
  reactNamespace: true,
  lexers: {
    jsx: ['JavascriptLexer', 'JsxLexer']
  },
  locales: ['en', 'de'],
  output: './locales'
};
```

The default behaviour is that `.js` file do not get parsed for React related code. Only `.jsx` file do that. That said I will update the code so that when `reactNamespace=true`, it is done automatically.

**Comment by dqunbp** [comment]
I try it in 1.0.0-beta4, but Trans key doesn't grabbing.

**Comment by karellm** [comment]
@dqunbp Did you try my other suggestion too? Does it work if you set the `lexers` option?

**Comment by karellm** [comment]
Fixed in `1.0.0-beta6`
